ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Java8/tasks/main.yml

```diff
@@ -16,7 +16,7 @@
 
 - name: Download Java 8
   win_get_url:
-    url: https://api.adoptopenjdk.net/v2/binary/releases/openjdk8?openjdk_impl={{ bootjdk }}&os=windows&arch=x64&release=latest&type=jdk&heap_size={{ heapsize }}
+    url: https://api.adoptopenjdk.net/v3/binary/latest/8/ga/windows/x64/jdk/{{ bootjdk }}/normal/adoptopenjdk?project=jdk
```

Comment:
Is changing behavior from heap size var to normal deliberate?

Reply:
It was as when `bootjdk = hotspot` and `heapsize = large`, the API returns nothing - however I've just seen that we don't have Large heapsize for hotspot, so that use-case wouldn't happen. I'll add it back in :)